CONTRIBUTING.md

```diff
@@ -141,6 +141,33 @@ to email the patch to <gap@gap-system.org>.  You can refer to
 [this tutorial](https://burzalodowa.wordpress.com/2013/10/05/how-to-send-patches-with-git-send-email/)
 on how to do this.
 
+## Reviewing code contributions
+
+When reviewing code, you can use the following list as a checklist.
+
+### Before you even run the code (see [here](https://lornajane.net/posts/2015/code-reviews-before-you-even-run-the-code))
+
+* Is it clear what feature / fix the contribution adresses?
+* Does it relate to exactly one issue?
+* Do the changed files look sensible? Use `git log --stat` to discover
```

Comment:
I find the sentence "Do the changed files look sensible?" rather unclear, I am not sure what it really is meant to say. E.g. is it meant to mean "Do the changes overall look fine?" or "Does the *list of* changed files look sensible?" ?  The next sentence suggest the second meaning could be intended. I suggest to rephrase it suitably.